jaeger-core/src/test/java/com/uber/jaeger/samplers/GuaranteedThroughputSamplerTest.java

```diff
@@ -48,8 +50,8 @@ public void testRateLimitingLowerBound() throws Exception {
     Assert.assertTrue(samplingStatus.isSampled());
     Map<String, Object> tags = samplingStatus.getTags();
 
-    assertEquals(tags.get(Constants.SAMPLER_TYPE_TAG_KEY), RateLimitingSampler.TYPE);
-    assertEquals(tags.get(Constants.SAMPLER_PARAM_TAG_KEY), 1.0);
+    assertEquals(tags.get(Constants.SAMPLER_TYPE_TAG_KEY), GuaranteedThroughputSampler.TYPE);
+    assertEquals(tags.get(Constants.SAMPLER_PARAM_TAG_KEY), 0.0001);
```

Comment:
Why do we return the probabilistic sampler's sampling rate even when the rate limiting lower bound sampler is triggered?